chromium/.eslintignore

```diff
@@ -0,0 +1 @@
+external/*
```

Comment:
We'll want to add this file to the top-level `.build_exclusions` as well.

Reply:
Done.